modules/n1ql/pages/n1ql-language-reference/update.adoc

```diff
@@ -407,3 +443,60 @@ RETURNING a;
 ]
 ----
 ====
+
+[[example-7]]
+.Update a document and set expiration
+====
+[.status]#Couchbase Server 6.5.1#
+
+Update a document and set the expiration to 1 week.
+
+.Query
+[source,n1ql]
+----
+UPDATE `travel-sample` t USE KEYS "route_10003"
+SET meta(t).expiration = 7*24*60*60,
+s.codeshare = NULL FOR s IN schedule END;
+----
+====
+
+[[example-8]]
+.Update a document and preserve expiration
+====
+[.status]#Couchbase Server 6.5.1#
+
+.Query
+[source,n1ql]
+----
+UPDATE `travel-sample` t USE KEYS "route_10003"
+SET meta(t).expiration = meta(t).expiration,
+s.codeshare = NULL FOR s IN schedule END;
+----
+====
+
+[[example-9]]
+.Update a document and unset expiration
+====
+[.status]#Couchbase Server 6.5.1#
+
+Set the document expiration to 0 to unset the document expiration.
+(In this case, the document expiration defaults to be the same as the bucket expiration.)
+
+.Query
+[source,n1ql]
+----
+UPDATE `travel-sample` t USE KEYS "route_10003"
+SET meta(t).expiration = 0,
+s.codeshare = NULL FOR s IN schedule END;
+----
+
+Alternatively, if you update the document without specifying the document expiration, the document expiration defaults to 0.
+
+.Query
+[source,n1ql]
+----
+UPDATE `travel-sample` t USE KEYS "route_10003"
+SET meta(t).expiration = 0,
```

Comment:
remove meta(t).expiration = 0, as u updating other fields this implicit as u said in line 493